pylint_django/transforms/foreignkey.py

```diff
@@ -16,6 +16,11 @@ def is_foreignkey_in_class(node):
     if not isinstance(node.parent.parent, ClassDef):
         return False
 
+    # Make sure the outfit class is the subclass of django.db.models.Model
+    is_in_django_model_class = node_is_subclass(node.parent.parent, 'Model')
```

Comment:
In other parts of the code we have:
`if not node_is_subclass(node, 'django.db.models.base.Model', '.Model')`

which should be more robust. You may want to look into that and update it.